pom.xml

```diff
@@ -230,12 +238,11 @@
 		</dependency>
 
 		<dependency>
-			<groupId>com.github.tomakehurst</groupId>
-			<artifactId>wiremock</artifactId>
-			<version>3.0.1</version>
+			<groupId>org.wiremock</groupId>
+			<artifactId>wiremock-standalone</artifactId>
+			<version>3.9.2</version>
```

Comment:
Currently not used as we removed the `CachingTestIT` but we will be trying to get this problematic package working here